Automattic/jetpack
# Dashboard: match dotcom's new default color scheme

#### Changes proposed in this Pull Request:

* Update color variables to match dotcom's new default color scheme in Jetpack Dashboard.

#### Testing instructions:

* Fire up this PR.
* Navigate through all Jetpack pages in wp-admin and ensure the base color scheme matches dotcom's default one.

#### Before

![image](https://user-images.githubusercontent.com/390760/52967314-9c819500-33a1-11e9-822b-6feeb83382e0.png)

![image](https://user-images.githubusercontent.com/390760/52967369-cc309d00-33a1-11e9-97d0-4a6a1e6ec954.png)

#### After

![image](https://user-images.githubusercontent.com/390760/52967326-a86d5700-33a1-11e9-8647-bde2baae3a5f.png)

![image](https://user-images.githubusercontent.com/390760/52967379-d6529b80-33a1-11e9-901e-4017f4822d46.png)

#### Proposed changelog entry for your changes:

* No changelog entry required, these are minor styling changes.
